src/app/Event.php

```diff
@@ -324,4 +327,8 @@ public function addTagById($tag)
         }
         return true;
     }
+
```

Comment:
Do you mean something like
```php
public scopeUngroupedInEvent(Builder $query, Event $event)
{
    $query->where('event_id', '==', $event->id)->where('event_ticket_group_id', '===', null);
}
```
in the `EventTicket`?

I think semantically it doesn't make much sense. I think asking the `Event` which of it's `EventTicket`s are not assigned any group is more intuitive than asking all `EventTicket`s which one belongs to an ``Event`` and also happens to be not assigned to a group

Also thinking about the usage of that method
https://github.com/Lan2Play/eventula-manager/blob/b5d4a94b997a51da5e4370da8056fbbc1659fff2/src/resources/views/admin/events/tickets/index.blade.php#L64
I don't know if using dynamic scopes would cause more problems there

Reply:
going to implement my proposal in https://github.com/Lan2Play/eventula-manager/pull/948, i also meaned a local scope, not a dynamic scope, had the wrong one in mind when writing this comment :D

Reply:
fyi: https://github.com/Lan2Play/eventula-manager/pull/948/commits/d4fa9df50172d1f2fb46068acfa6531557b558c2